packages/runtime/container-runtime-definitions/package.json

```diff
@@ -93,10 +93,15 @@
 	"typeValidation": {
 		"broken": {
 			"InterfaceDeclaration_IContainerRuntime": {
-				"backCompat": false
+				"backCompat": false,
+				"forwardCompat": false
 			},
 			"InterfaceDeclaration_IContainerRuntimeWithResolveHandle_Deprecated": {
-				"backCompat": false
+				"backCompat": false,
+				"forwardCompat": false
+			},
+			"TypeAliasDeclaration_IContainerRuntimeBaseWithCombinedEvents": {
```

Comment:
Is it okay to break these in main now? (I am unsure what our rules are for breaking changes)

Reply:
As we discussed in our team's stand-up with internal major's gone, we have no other vehicle than minor releases to deliver changes to alpha APIs.